DataStructures/BitArray/TestBitArray.cs

```diff
@@ -0,0 +1,377 @@
+﻿/**
+ * Author: Christian Bender
+ * Class: TestBitArray
```

Comment:
Please move this file to the `UnitTest` projects. Test files are not packaged with the source files of Data Structures and Algorithms.

Reply:
Should I add only the class BitArray without tests?